docs/sources/write/front-matter/index.md

```diff
@@ -288,7 +288,11 @@ cascade:
 
 <!-- vale Grafana.Headings = YES -->
 
-Use `menuTitle` to specify a different heading in the sidebar navigation than the `title` element; for example, if you want to abbreviate the topic heading in the table of contents.
+Use `menuTitle` to specify a different heading in the sidebar navigation than the `title` element.
+For example, if you want to abbreviate the topic heading in the table of contents.
+
+Don't remove the verb from [task topic](https://grafana.com/docs/writers-toolkit/structure/topic-types/task/) headings.
```

Comment:
I'm not sure why this was included here. It seems pretty tangential to the `menuTitle` element.

Reply:
This is actually the crux of the change so I think I need to reword it. When deriving the menuTitle from the title, you must keep the verb to indicate that the page is a task before the reader clicks on it. I think the use of heading is making this more confusing?

Reply:
I've made the change to use 'title' everywhere. Does it make more sense now?